pom.xml

```diff
@@ -37,10 +44,8 @@
     <version.assertj>3.5.2</version.assertj>
     <version.jcip.annotations>1.0</version.jcip.annotations>
     <version.shrinkwrap.resolver>2.2.5</version.shrinkwrap.resolver>
+    <version.snakeyaml>1.17</version.snakeyaml>
 
-    <!-- override from parent -->
-    <maven.compiler.target>1.5</maven.compiler.target>
-    <maven.compiler.source>1.5</maven.compiler.source>
```

Comment:
I think it was here for a reason. We support older containers which are not running on JDK 8. If we compile to JDK 8 bytecode it will make Chameleon unusable for those users. I think we have to keep it like that until we decide when to drop the support for those legacy one.